### PR TITLE
fixed locale pt-br not being used and added extra suport for pt-br

### PR DESCRIPTION
**Pull request by pedropacheco92**

### CODE REVIEW

**Comment by joaopedsa** [comment]
Nice Work!!

File: packages/component/src/Localization/Localize.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ function normalizeLanguage(language) {
   } else if (language.startsWith('pl')) {
     return 'pl-PL';
   } else if (language.startsWith('pt')) {
-    if (language === 'pt-BR') {
+    if (language === 'pt-br') {
```
Comment (corinagum) [inline packages/component/src/Localization/Localize.js]: could you explain why this is necessary?

File: packages/component/src/Localization/pt-BR.js [hunk 2]
```diff
@@ -1,3 +1,44 @@
...
+      hour12: false,
+      minute: '2-digit',
+      month: '2-digit',
+      year: 'numeric',
+    });
+  }
+}
+
 export default {
   // FAILED_CONNECTION_NOTIFICATION: '',
```
Comment (corinagum) [inline packages/component/src/Localization/pt-BR.js]: Are you able to fill the commented-out lines to finish up this localization file? :)

File: packages/component/src/Localization/Localize.js (hunk 1 above)
Comment (pedropacheco92) [reply packages/component/src/Localization/Localize.js]: in the first line of this function, the language is setted to lower case, and this condidion was never true.

File: packages/component/src/Localization/pt-BR.js (hunk 2 above)
Comment (pedropacheco92) [reply packages/component/src/Localization/pt-BR.js]: Sure!

File: packages/component/src/Localization/Localize.js (hunk 1 above)
Comment (corinagum) [reply packages/component/src/Localization/Localize.js]: Ah I understand now, thank you!

**Comment by pedropacheco92** [comment]
Fixed conflict

**Comment by corinagum** [comment]
Will merge in once tests pass. Thanks again!

**Comment by corinagum** [comment]
These changes will be in `/latest/` of the 4.4 release. :)

**Pull request merged**